fcrepo-clients/src/main/java/edu/unc/lib/dl/event/FilePremisLogger.java

```diff
@@ -170,11 +170,13 @@ public Model getModel() {
 			// Construct the event object with a presupplied model
 			PremisEventObject event = new PremisEventObject(eventPid, repository,
 					repository.getRepositoryObjectDataLoader());
-			event.storeModel(model);
+			event.storeModel(eventModel);
 
 			events.add(event);
 		}
 
+		log.debug("Retrieved {} events from file log for object {}",
```

Comment:
no, confusingly it works differently for log4j loggers versus the java string formatting method, which is used by the failJob method.